definitions/contested/json/CaseTypeTab/CaseTypeTab-nonprod.json

```diff
@@ -7,5 +7,14 @@
     "TabDisplayOrder": 7,
     "CaseFieldID": "generalLetterCollection",
     "TabFieldDisplayOrder": 17
+  },
+  {
+    "LiveFrom": "01/01/2021",
+    "CaseTypeID": "FinancialRemedyContested",
+    "TabID": "applicationDocumentsTab",
```

Comment:
same as above, don't use generic nonprod files. I think your ticket should be toggled on.